src/Application/Grouper/MethodGrouper.php

```diff
@@ -65,7 +65,7 @@ public function groupMethods(array $methods): array
                 $getMethod = \str_replace('set', 'get', $method);
                 if (isset($methods[$getMethod])) {
                     $type = 'get_set';
-                    $name = 'testSetGet' . \substr($method, 3);
+                    $name = 'test' . \substr($method, 3);
```

Comment:
same here this is testing currently `SetGet` and that is why it is named this way. It should be as explicit as possible as there could be other methods for that attribute.